src/Compatibility/Microsoft.DotNet.PackageValidation/Helpers.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.IO.Compression;
+
+namespace Microsoft.DotNet.PackageValidation
+{
+    internal class Helpers
+    {
+        public static Stream GetFileStreamFromPackage(string packagePath, string entry)
+        {
+            FileStream stream = File.OpenRead(packagePath);
+            var zipFile = new ZipArchive(stream);
+            MemoryStream ms = new MemoryStream();
```

Comment:
Should we keep track of the `MemoryStreams` that we are creating and dispose them at the end of the execution?